components/home/features/details/EvmProtocols.vue

```diff
@@ -0,0 +1,122 @@
+<script setup lang="ts">
+import { get } from '@vueuse/core';
+import { useIntegrationsData } from '~/composables/integrations';
+import type { IntegrationItem } from '~/types/integrations';
+
+const { t } = useI18n();
+
+// Sorted by TVL, source: Defillama
+const biggestProtocols: string[] = [
```

Comment:
any way to avoid hardcoding them and make it easier to update?

Reply:
Yeah, the issue here is that the metadata we have for the protocol does not match the identifier of DefiLlama.

Probably it's still better to hardcode the mapping between our protocol and the identifier of Defillama, and then fetch the TVL through endpoint

Reply:
Ok, so we would still need to do some manual work. Let's move it to a next step then, in a follow up to this. 

What we need is to be able to updated the order along with the integrations, with the least amount of effort